packages/components/src/overlays/Popover/src/Popover.module.css

```diff
@@ -0,0 +1,92 @@
+:where(.hop-Popover *) {
+    font-family: var(--font-family);
+    font-size: var(--font-size);
+    font-weight: var(--font-weight);
+    line-height: var(--line-height);
+    color: var(--color);
+
+    outline: none;
+}
+
+.hop-Popover__dialog {
+    --hop-Popover-background: var(--hop-neutral-surface);
+    --hop-Popover-font-family: var(--hop-body-sm-font-family);
+    --hop-Popover-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-font-weight: var(--hop-body-sm-font-weight);
+    --hop-Popover-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-color: var(--hop-neutral-text);
+    --hop-Popover-title-font-family: var(--hop-body-sm-font-family);
+    --hop-Popover-title-font-weight: var(--hop-body-sm-semibold-font-weight);
+    --hop-Popover-content-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-content-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-content-color: var(--hop-neutral-text-weak);
+    --hop-Popover-footer-font-size: var(--hop-body-sm-font-size);
+    --hop-Popover-footer-line-height: var(--hop-body-sm-line-height);
+    --hop-Popover-footer-link-color: var(--hop-primary-text);
+    --hop-Popover-border: 0.0625rem solid var(--hop-neutral-border-weak);
+    --hop-Popover-border-radius: var(--hop-shape-rounded-md);
+    --hop-Popover-box-shadow: var(--hop-elevation-lifted);
+    --hop-Popover-max-inline-width: 25rem;
+
+    /* Internal variables */
+    --font-family: var(--hop-Popover-font-family);
+    --font-size: var(--hop-Popover-font-size);
+    --font-weight: var(--hop-Popover-font-weight);
+    --line-height: var(--hop-Popover-line-height);
+
+    display: grid;
+    grid-template-areas:
+        "header header"
+        "content content"
+        "footer action";
+    grid-template-columns: repeat(2, auto);
+    gap: var(--hop-space-inset-sm);
+
+    box-sizing: border-box;
+    inline-size: var(--hop-Popover-max-inline-width);
+    max-inline-size: var(--hop-Popover-max-inline-width);
+    padding: var(--hop-space-inset-md);
+
+    background-color: var(--hop-Popover-background);
+    border: var(--hop-Popover-border);
+    border-radius: var(--hop-Popover-border-radius);
+    box-shadow: var(--hop-Popover-box-shadow);
+}
+
+.hop-Popover__title {
+    --font-family: var(--hop-Popover-title-font-family);
+    --font-weight: var(--hop-Popover-title-font-weight);
+    --color: var(--hop-Popover-color);
+
+    grid-area: header;
+}
+
+.hop-Popover__actions,
+.hop-Popover__action {
+    grid-area: action;
+    justify-self: end;
+    margin-block-start: var(--hop-space-stack-sm);
+}
+
+.hop-Popover__title + :where(p, span, div) {
```

Comment:
Can you explain why this selector exists?

Reply:
Since we don't have a content wrapper, the only way to style content is to use this type of pointer. 

the `+` means that the element following the class is the element to be pointed to.

Example: https://webdesign.tutsplus.com/the-30-css-selectors-you-must-memorize--net-16048t#toc-qa4w-x-y

Reply:
@alexasselin008 did this explanation help you understand the reason for this selector?

Reply:
I understand, but i think we should use a Content component to properly position and style the content. Just like in Orbiter 

```
<PopoverTrigger>
    <Button variant="secondary">Trigger</Button>
    <Popover>
        <Heading>WikiSpace</Heading>
        <Content>
            SpaceX designs, manufactures, and launches the world's most advanced rockets and
            spacecraft. The company was founded in 2002 by Elon Musk to revolutionize space
            transportation, with the ultimate goal of making life multi-planetary.
        </Content>
    </Popover>
</PopoverTrigger>
```

Spectrum uses the same strategy: 
https://react-spectrum.adobe.com/react-spectrum/Content.html

We'll need the Content component for multiple other components, such as Modal, IllustratedMessage, Cards. 

having the + selector is not "safe" enough. For instance, if i wanted a popover with two <section> component